examples/nlp/language_modeling/tuning/megatron_gpt_peft_eval.py

```diff
@@ -83,9 +83,9 @@ def main(cfg) -> None:
         gradient_as_bucket_view=cfg.model.gradient_as_bucket_view,
         find_unused_parameters=False,
     )
-    if cfg.trainer.precision in [16, "bf16"]:
+    if cfg.trainer.precision in [16, "bf16", '16-mixed', 'bf16-mixed']:
         scaler = None
-        if cfg.trainer.precision == 16:
+        if cfg.trainer.precision == 16 or cfg.trainer.precision == '16-mixed':
```

Comment:
nit: why not `if cfg.trainer.precision in [16, '16', '16-mixed]` ? This is less characters and it's similar to what we do in line 86. Also 16 should probably be `[16, '16', 'bf16', ...`]

Reply:
Thanks @ericharper, this is added in the latest commit.